tests/integration/cairo_compiler/contracts_compilation_test.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+import pytest
+
+import cairo_python_bindings
+
+from tests.data.cairo1_contracts import (
+    CAIRO_BINDINGS_CONTRACT_STARKNET_HELLO,
+    CAIRO_BINDINGS_CONTRACT_STARKNET_HELLO_TEST,
+)
+from tests.integration.conftest import CreateProtostarProjectFixture
+from tests.integration._conftest import ProtostarFixture
+
+
+@pytest.fixture(name="protostar")
+def protostar_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar:
+        protostar.build_sync()
+        yield protostar
+
+
+@pytest.fixture(name="prepared_files")
+def prepared_files_fixture(protostar: ProtostarFixture):
+    cairo_hello_path = Path("./src/hello.cairo")
+    cairo_test_path = Path("./src/test.cairo")
+    casm_path = Path("./src/example.casm")
+
+    paths = {
+        "cairo_hello": cairo_hello_path,
+        "cairo_test": cairo_test_path,
+        "casm": casm_path,
+    }
+
+    for _, path in paths.items():
+        protostar.create_files({str(path): ""})
+        assert path.exists()
+
+    with open(cairo_hello_path, "w") as file:
+        file.write(CAIRO_BINDINGS_CONTRACT_STARKNET_HELLO)
+
+    with open(cairo_test_path, "w") as file:
+        file.write(CAIRO_BINDINGS_CONTRACT_STARKNET_HELLO_TEST)
+
+    return paths
```

Comment:
no need for this logic

Reply:
[extract fixture to a sep file](https://github.com/software-mansion/protostar/pull/1316/commits/25d34a76296e38402292118940d46c8917b2c850)